Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -137,7 +137,11 @@ class RUMViewScopeTests: XCTestCase {
         let custonContext: DatadogContext = .mockWith(source: source)
 
         var context = self.context
-        context.launchTime = .init(launchTime: 2, isActivePrewarm: false)
+        context.launchTime = .init(
+            launchTime: 2,
+            launchDate: nil,
+            isActivePrewarm: false
+        )
```

Comment:
Is this relevant for `testWhenConfigurationSourceIsSet_applicationStartUsesTheConfigurationSource`? The `context.launchTime` doesn't seem to be considered for sending "app start" action so we should be able to remove the entire `context` var, no 🤔?
https://github.com/DataDog/dd-sdk-ios/blob/fbc11e4af4f2d83b891fe8177d84db1e25e663a3/Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift#L145-L149

Reply:
Indeed 👍